backend/dataall/modules/omics/cdk/pivot_role_omics_policy.py

```diff
@@ -0,0 +1,37 @@
+from aws_cdk import aws_iam as iam
+from dataall.core.environment.cdk.pivot_role_stack import PivotRoleStatementSet
+
+
+class OmicsPolicy(PivotRoleStatementSet):
+    """
+    Creates an Omics policy for Pivot role accessing and interacting with Omics Projects
+    """
+
+    def get_statements(self):
+        return [
+            iam.PolicyStatement(sid='OmicsList', actions=['omics:List*'], resources=['*']),
+            iam.PolicyStatement(
+                sid='OmicsWorkflowActions',
+                actions=['omics:GetWorkflow'],
+                resources=[
+                    f'arn:aws:omics:{self.region}:{self.account}:workflow/*',
+                    f'arn:aws:omics:{self.region}::workflow/*',
+                ],
+            ),
+            iam.PolicyStatement(
+                sid='PassRoleOmics',
+                actions=[
+                    'iam:PassRole',
+                ],
+                resources=[
+                    f'arn:aws:iam::{self.account}:role/{self.env_resource_prefix}*',
+                ],
+                conditions={
+                    'StringEquals': {
+                        'iam:PassedToService': [
+                            'omics.amazonaws.com',
+                        ]
+                    }
+                },
+            ),
```

Comment:
With the current implementation the pivot role still is the role used to start, delete and get run information as well as get workflow information in omics

If we wanted to switch to have a particular env role do it we would need to make a few changes on OmicsClient to assume into a specific and go from there.

I am comfortable considering that as an enhancement to this feature, and with that being said we are just missing some permissions we need to add back to pivot role, more specifically we need all of the permissions listed below defined as CDK in this return:

```
        {
            "Action": [
                "omics:GetWorkflow",
                "omics:ListWorkflows",
                "omics:StartRun",
                "omics:TagResource"
            ],
            "Resource": [
                "arn:aws:omics:{REGION}:{ACCOUNT}:workflow/*",
                "arn:aws:omics:{REGION}:workflow/*",
                "arn:aws:omics:{REGION}:{ACCOUNT}:run/*"
            ],
            "Effect": "Allow",
            "Sid": "OmicsWorkflowActions"
        },
        {
            "Action": [
                "omics:ListRuns",
                "omics:DeleteRun",
                "omics:GetRun",
                "omics:ListRunTasks",
                "omics:CancelRun"
            ],
            "Resource": [
                "arn:aws:omics:{REGION}{ACCOUNT}:run/*"
            ],
            "Effect": "Allow",
            "Sid": "OmicsRunActions"
        },
        {
            "Condition": {
                "StringEquals": {
                    "iam:PassedToService": [
                        "omics.amazonaws.com"
                    ]
                }
            },
            "Action": "iam:PassRole",
            "Resource": "arn:aws:iam::{ACCOUNT}:role/dataall*",
            "Effect": "Allow",
            "Sid": "PassRoleOmics"
        },
```

Reply:
I see; with that in mind, I went ahead and reverted back my previous change to allow the pivot role to perform the above actions again. I think post-release, we should clean that up to move CRUD operations to the env role rather than pivot. but it is ok for now like you mentioned.